hack/testing/rsyslog/playbook.yaml

```diff
@@ -97,153 +45,13 @@
-    when:
-    - elasticsearch_ops_server_host is not defined
-    - openshift_logging_use_ops | default(False)
-
-  - name: install template config files
-    template: src={{ item }}.j2 dest=/etc/rsyslog.d/viaq/{{ item }} mode=0400
-    with_items:
-    - elasticsearch.conf
-
-  - name: install config files
-    copy: src={{ item }} dest=/etc/rsyslog.d/viaq/{{ item }} mode=0400
-    with_items:
-    - normalize_level.json
-    - prio_to_level.json
-    - viaq_formatting.conf
-    - k8s_filename.rulebase
-    - k8s_container_name.rulebase
-    - parse_json.rulebase
-
-  - name: install main viaq config file
-    copy: src={{ item }} dest=/etc/rsyslog.d/{{ item }} mode=0400
-    with_items:
-    - viaq_main.conf
-
-  - name: copy selinux policy files
-    copy: src={{ item }} dest=/root/ mode=0400
-    with_items:
-    - rsyslog2k8s.te
-    - rsyslog2es.te
-    - varlibdockercont.te
-    - rsyslogaccessnssdb.te
-    - rsyslogvar.te
-
-  - name: build policy
-    shell: |
-      cd /root
-      sysmods=$( semodule -l | awk '{print $1}' )
-      for mod in varlibdockercont rsyslog2k8s rsyslogaccessnssdb rsyslog2es rsyslogvar ; do
-        if echo "$sysmods" | grep -q $mod ; then
-          echo using existing selinux module $mod
-        elif [ -f ${mod}.te ] ; then
-          checkmodule -M -m -o ${mod}.mod ${mod}.te
-          semodule_package -o ${mod}.pp -m ${mod}.mod
-          semodule -i ${mod}.pp
-        fi
-      done
-
-  # openshift devenv is very noisy - need to increase rsyslog limits for imjournal
-  - name: Update rsyslog.conf imjournal
-    lineinfile:
-      dest: /etc/rsyslog.conf
-      regexp: '^(\$ModLoad imjournal|module\(load="imjournal")'
-      line: 'module(load="imjournal" StateFile="imjournal.state" UsePidFromSystem="on" RateLimit.Burst="{{ imjournal_ratelimit_burst|default(1000000) }}" RateLimit.Interval="{{ imjournal_ratelimit_interval|default(10) }}" PersistStateInterval="1000")'
-      backup: yes
-
-  - name: Comment out rsyslog.conf IMJournalStateFile
-    lineinfile:
-      dest: /etc/rsyslog.conf
-      regexp: '^\$IMJournalStateFile|^#\$IMJournalStateFile'
-      line: '#$IMJournalStateFile imjournal.state'
-
-  - name: disable rsyslog
-    systemd:
-      name: rsyslog.service
-      enabled: no
-      state: stopped
-    when: use_rsyslog_image | default(False)
-
-  - name: install rsyslog container systemd service
-    template: src=rsyslog-container.service.j2 dest=/usr/lib/systemd/system/rsyslog-container.service mode=0400
-    when: use_rsyslog_image | default(False)
-
-  - name: enable and run rsyslog-container.service
-    systemd:
-      name: rsyslog-container.service
-      daemon_reload: yes
-      enabled: yes
-      state: started
-    when: use_rsyslog_image | default(False)
-
-  - name: reload and restart rsyslog.service
-    systemd:
-      name: rsyslog.service
-      daemon_reload: yes
-      enabled: yes
-      state: restarted
-    when: not use_rsyslog_image | default(False)
+  tasks:
```

Comment:
you don't need `tasks` here since this is already part of the task list of the play `- name: install and configure rsyslog on the nodes` - you are getting the following error: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_origin-aggregated-logging/1314/test_pull_request_origin_aggregated_logging_json_file/1021/artifacts/scripts/test-zzz-rsyslog/artifacts/zzz-rsyslog-ansible.log
```
 [WARNING]: While constructing a mapping from /data/src/github.com/openshift
/origin-aggregated-logging/hack/testing/rsyslog/playbook.yaml, line 32, column
3, found a duplicate dict key (tasks). Using last defined value only.
```
not sure where the `roles` should go, before or after the list of tasks